clients/graphql-kotlin-client-jackson/build.gradle.kts

```diff
@@ -7,6 +7,8 @@ plugins {
 dependencies {
     api(projects.graphqlKotlinClient)
     api(libs.jackson)
+    testImplementation("org.junit.jupiter:junit-jupiter")
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
```

Comment:
same as above -> define it in toml file and pull it in buildsrc

@samuelAndalon any objections on adding this extra dep? it just helps with running stuff from intellij (but potentially could be skipped)